Marlin/src/lcd/language/language_fr.h

```diff
@@ -113,7 +113,7 @@ namespace Language_fr {
   PROGMEM Language_Str MSG_SWITCH_PS_ON                    = _UxGT("Allumer alim.");
   PROGMEM Language_Str MSG_SWITCH_PS_OFF                   = _UxGT("Eteindre alim.");
   PROGMEM Language_Str MSG_EXTRUDE                         = _UxGT("Extrusion");
-  PROGMEM Language_Str MSG_RETRACT                         = _UxGT("Retrait");
+  PROGMEM Language_Str MSG_RETRACT                         = _UxGT("Rétractation");
```

Comment:
This is not french

Reply:
Not in this case , because MSG RETRACT , is the name of a submenu 👍

Reply:
Tu es francais ?